packages/react/src/ReactLazy.js

```diff
@@ -8,13 +8,52 @@
 import type {LazyComponent, Thenable} from 'shared/ReactLazyComponent';
 
 import {REACT_LAZY_TYPE} from 'shared/ReactSymbols';
+import warning from 'shared/warning';
 
 export function lazy<T, R>(ctor: () => Thenable<T, R>): LazyComponent<T> {
-  return {
+  let lazyType = {
     $$typeof: REACT_LAZY_TYPE,
     _ctor: ctor,
     // React uses these fields to store the result.
     _status: -1,
     _result: null,
   };
+
+  if (__DEV__) {
+    // In production, this would just set it on the object.
```

Comment:
I guess it would be enumerable in prod but not in dev? prolly ok tho

Reply:
I figured it's bad regardless but can change

Reply:
fixed